### ISSUE TITLE
error with interactive mode in version 9.0.0-BETA

### ISSUE THREAD

**Issue by 111andre111** [elastic/support-diagnostics#452]
With the new version 9.0.0 and a local elasticsearch cluster this one works:
`./diagnostics.sh -url https://localhost:9200`

But when do the same using the interactive mode, it ends up with an error:
```
Command line options can be displayed with the --help argument. Entering interactive mode.
Enter the number of the diagnostic type to run.
* 1: local - Node on the same host as the diagnostic utility.
  2: remote - Node on a different host than the diagnostic utility
  3: api - Elasticsearch REST API calls, no system calls or logs.
  4: cloud - Cloud cluster = only Elasticsearch REST API calls.
  5: logstash-local - Logstash process on the same host as the diagnostic utility.
  6: logstash-remote - Logstash on a different host than the diagnostic utility.
  7: logstash-api - Logstash REST calls. No system calls.
Enter your choice: 3
Bypass the diagnostic version check. Use when internet outbound HTTP access is blocked by a firewall.  (Y/N) [N]:
 Required field. Full uri in format <scheme>://<host>:<port> ex. https://localhost:9200.  HTTP access for this node must be enabled.  https://localhost:9200
Attempt to retry failed REST calls?  (Y/N) [N]:
Fully qualified path to an output directory. If it does not exist the diagnostic will attempt to create it. If not specified the diagnostic directory will be used:  [/Users/andreletterer/Downloads/support-diagnostics-9.0.0]:
Creating temp directory: /Users/andreletterer/Downloads/support-diagnostics-9.0.0/diagnostics-temp
Fatal error occurred: Error setting up http connection for: null. Check diagnostics.log in the archive file for more detail.
```

**Comment by jpcarey** [comment]
thanks for reporting. @nemonster any thoughts on this?

**Comment by nemonster** [comment]
@jpcarey Looking into it, and will have something a little later today.

**Comment by nemonster** [comment]
@111andre111 @jpcarey 

TLDR: The currently attached archive should be working.

Minor issue in url validation slipped through after a late refactor. I've updated the README for the beta to reflect that it's changed but just replaced the zip and SHA rather than doing a full release this time. We can revisit if anything more substantive pops up.

**Comment by 111andre111** [comment]
Cool, thanks. It's working.
